.eslintrc.json

```diff
@@ -33,7 +33,15 @@
         "react/react-in-jsx-scope": "off",
         "react/jsx-filename-extension": [1, { "extensions": [".tsx", ".jsx"] }],
         "react/jsx-wrap-multilines": ["error", {"declaration": "parens-new-line", "assignment": "parens-new-line", "return": "parens-new-line", "arrow": "parens-new-line", "condition": "parens-new-line", "logical": "parens-new-line", "prop": "parens-new-line"}],
-        "import/prefer-default-export": "off"
+        "import/prefer-default-export": "off",
+        "react/require-default-props": "off"
+    //     "react/function-component-definition": [
```

Comment:
Why whis rules are commented? And why this rules are need?

Reply:
This is an optional rule. We can decide in which form we will write functions

Reply:
You can change or delete this rule. Never use comments for change rules. And always explain why rule should be changed/deleted.